Annex60/ThermalZones/ReducedOrder/Validation/VDI6007/package.order

```diff
@@ -1,7 +1,4 @@
 TestCase1
-TestCase10
```

Comment:
Nice to have that again in the correct order, but the Merge-Script of `BuildingsPy` will resort that.